sentences/de/cover_HassGetState.yaml

```diff
@@ -1,4 +1,46 @@
 language: de
 intents:
   HassGetState:
-    data: []
+    data:
+      - sentences:
+          - "ist <name> {cover_states:state} [<area>]"
+          - "ist <name> [<area>] {cover_states:state}"
+          - "ist [<area>] <name> {cover_states:state}"
+          - "{cover_states:state} [sich] <name> [<area>]"
+        response: einzeln_janein
+        requires_context:
+          domain: cover
+        slots:
+          domain: cover
+
+      - sentences:
+          - "sind [irgendwelche | einige] {cover_classes:device_class} {cover_states:state} [<area>]"
+          - "sind [irgendwelche | einige] {cover_classes:device_class} [<area>] {cover_states:state}"
```

Comment:
here whitespaces before/after | should be removed in order not to lead to double whitespaces

Reply:
even whitespaces were normalized during matching (_reduced to single word delimiter_), i also prefer to have a unique writing of these :+1

Reply:
- #1135